src/io/mpkarchive.cpp

```diff
@@ -80,12 +80,17 @@ IoError MpkArchive::Create(InputStream* stream, VfsArchive** outArchive) {
   for (uint32_t i = 0; i < FileCount; i++) {
     uint32_t Compression = ReadLE<uint32_t>(stream);
     uint32_t Id = ReadLE<uint32_t>(stream);
+
```

Comment:
If we have an empty line here, let's add one before the duplicate if check as well, seems appropriate to me imo